animation_nodes/ui/node_menu.py

```diff
@@ -370,6 +370,9 @@ def draw(self, context):
         insertNode(layout, "an_VertexGroupInputNode", "Vertex Group Input")
         insertNode(layout, "an_MeshFromSplineNode", "Mesh From Spline")
         layout.separator()
+        insertNode(layout, "an_VertexGroupInputNode", "Vertex Group Input")
```

Comment:
`an_VertexGroupInputNode` is already inserted above.

Reply:
I have put the _Vertex Group Input_ and _Set Vertex Weights_ nodes into a separate category.